packages/core/test-helpers/core-test-helpers-deprecations-getters/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "@kbn/core-test-helpers-deprecations-getters",
+  "private": true,
+  "version": "1.0.0",
+  "main": "./target_node/index.js",
```

Would you mind adding the author to the package?
```suggestion
  "main": "./target_node/index.js",
  "author": "Kibana Core",
```
Refer to https://github.com/elastic/kibana/pull/137887 for the reasoning.